### ISSUE TITLE
NPM install

### ISSUE THREAD

**Issue by bpalermo** [jsmodules/angular-material-calendar#38]
Error while installing with command:

```
npm install angular-material-calendar --save
```

```
npm ERR! angular-material-calendar@0.2.12 postinstall: `./node_modules/protractor/bin/webdriver-manager update`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the angular-material-calendar@0.2.12 postinstall script './node_modules/protractor/bin/webdriver-manager update'.
npm ERR! This is most likely a problem with the angular-material-calendar package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/protractor/bin/webdriver-manager update
npm ERR! You can get their info via:
npm ERR!     npm owner ls angular-material-calendar
npm ERR! There is likely additional logging output above.
```

**Comment by gavinbarron** [comment]
I assume you're running a Windows machine?

This is a known issue with protractor on Windows. I've found it necessary to cd into the node_modules\protractor\bin dir and run node webdriver-manager update manually to update protractor.

**Comment by bpalermo** [comment]
Actually I`m running Linux Ubuntu 14.04 LTS

**Comment by gavinbarron** [comment]
Did manually running the command work as expected?

**Comment by bpalermo** [comment]
I think so:

```
$ node webdriver-manager update
Updating selenium standalone
downloading https://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar...
Updating chromedriver
downloading https://chromedriver.storage.googleapis.com/2.19/chromedriver_linux64.zip...
chromedriver_2.19.zip downloaded to /home/palermo/Projetos/sanus/frontend/node_modules/protractor/selenium/chromedriver_2.19.zip
selenium-server-standalone-2.47.1.jar downloaded to /home/palermo/Projetos/sanus/frontend/node_modules/protractor/selenium/selenium-server-standalone-2.47.1.jar
```

**Comment by gavinbarron** [comment]
That looks correct to me....
@bradberger Got any ideas here?

**Comment by bradberger** [comment]
Hmm, the script actually uses the locally installed selenium instance, in the relative path of the project, so `./node_modules/protractor/bin/webdriver-manager`. Obviously, the `ENOENT` error means that file doesn't exist, but it should since it is included in `package.json`. Unless it's moved in a newer version of protractor? 

I'm checking it out on my machine, since it's also Ubuntu 14.04, we'll see what we can figure out.

**Comment by bradberger** [comment]
Seems like it's a mistake in the `package.json`. Protractor is only required for development, so it really shouldn't be required when installing for 99% of people who aren't actually working with developing the source itself. So since it's only part of dev dependencies, Protractor isn't getting installed, and the file doesn't exist.

Short term solution would be for you to add `protractor` to your `package.json`, but obviously we'll just need to remove the `postinstall` script from this package's `package.json`. 

So in short it's a bug and we'll get it fixed.

**Comment by bradberger** [comment]
It's fixed with c9987d4, just make sure that you're using the updated version for install now:

```json
{
  "dependencies": {
    "angular-material-calendar": "^0.2.14"
  }
}
```

**Comment by bpalermo** [comment]
Fixed thanks!
